src/lib/orion_websocket/parser.cpp

```diff
@@ -30,10 +30,19 @@
 #include "rapidjson/writer.h"
 
 #include "logMsg/logMsg.h"
+#include "rest/HttpHeaders.h"
 
-void ws_parser_parse(std::string &url, std::string &verb, std::string &payload, const char *msg)
+
+#include <map>
+
+
+void ws_parser_parse(const char *msg,
+                     std::string &url, std::string &verb,
+                     std::string &payload, HttpHeaders &head)
 {
```

Comment:
More style guide:

With long arg lists we normally (me, always) lay them out like this:

    void ws_parser_parse
    (
      const char*   msg,
      std::string&  url,
      std::string&  verb,
      std::string&  payload,
      HttpHeaders&  head
    )
    {
      ...
    }

It's easier to read that way.

Note that '\*' and '&' are 'glued' to the **type**, not the *variable*.
This makes a lot of sense as these electors change the type (*this* is my idea :-))
Also note that there are two spaces between type and variable name, not one.
This is due to some type need more than one word, e.g:

    struct timeval  tv;

All style guide stuff ... we have a lot of rules ...